packages/web-components/fast-foundation/src/carousel/carousel.spec.md

```diff
@@ -48,67 +50,79 @@ As defined by the W3C:
 - `aria-labelledby` - optional based on implementation**
 - `aria-label` - optional, based on implementation**
 - `paused` - boolean whether the rotation is paused or not
-- `activeslideid` - string
+- `active-slide-id` / `activeid` - string, active id of slide or tab depending on pattern
+- `basic-pattern` - boolean whether or not the carousel follows the basic or tabbed pattern per ARIA spec, defaults to tabbed when basic-pattern is false (See the [ARIA Carousel Patterns](https://w3c.github.io/aria-practices/#basic-carousel-elements))
```

Comment:
there's also a "group" pattern, should we future proof this by making it a string instead? Such that:
```html
<fast-carousel pattern="basic"></fast-carousel>
```